packages/svelte/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@sentry/svelte",
+  "version": "7.9.0",
+  "description": "Official Sentry SDK for Svelte",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/svelte",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
+  "main": "build/cjs/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/types/index.d.ts",
+  "private": "true",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/browser": "7.9.0",
+    "@sentry/types": "7.9.0",
+    "@sentry/utils": "7.9.0",
+    "tslib": "^1.9.3"
+  },
+  "peerDependencies": {
+    "svelte": "3.x"
+  },
+  "scripts": {
+    "build": "run-p build:rollup build:types",
+    "build:dev": "run-s build",
+    "build:rollup": "rollup -c rollup.npm.config.js",
+    "build:types": "tsc -p tsconfig.types.json",
+    "build:watch": "run-p build:rollup:watch build:types:watch",
+    "build:dev:watch": "run-s build:watch",
+    "build:rollup:watch": "rollup -c rollup.npm.config.js --watch",
+    "build:types:watch": "tsc -p tsconfig.types.json --watch",
+    "build:npm": "ts-node ../../scripts/prepack.ts && npm pack ./build",
+    "circularDepCheck": "madge --circular src/index.ts",
+    "clean": "rimraf build coverage sentry-svelte-*.tgz",
+    "fix": "run-s fix:eslint fix:prettier",
+    "fix:eslint": "eslint . --format stylish --fix",
+    "fix:prettier": "prettier --write \"{src,test,scripts}/**/*.ts\"",
+    "lint": "run-s lint:prettier lint:eslint",
+    "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
+    "lint:prettier": "prettier --check \"{src,test,scripts}/**/*.ts\"",
+    "test": "jest --passWithNoTests",
```

Comment:
Just added the `--passWithNoTests` flag so that CI passes until we actually have tests